src/Bridge/Laravel/DbToolsServiceProvider.php

```diff
@@ -0,0 +1,270 @@
+                }
+            }
+        );
+
+        $this->app
+            ->when(AnonymizerRegistry::class)
+            ->needs('$paths')
+            ->give(function (Application $app) {
+                // Validate user-given anonymizer paths.
+                $anonymizerPaths = $app->make('config')->get('db-tools.anonymizer_paths');
+                foreach ($anonymizerPaths as $path) {
+                    if (!\is_dir($path)) {
+                        throw new \InvalidArgumentException(\sprintf(
+                            '"db_tools.anonymizer_paths": path "%s" does not exist', $path
+                        ));
+                    }
+                }
+                // Set the default anonymizer directory only if the folder
+                // exists in order to avoid "directory does not exist" errors.
+                $defaultDirectory = $app->basePath('app/Anonymizer');
+                if (\is_dir($defaultDirectory)) {
+                    $anonymizerPaths[] = $defaultDirectory;
+                }
+            })
+        ;
+
+        /*$this->app->singleton(BackupperFactory::class, function (Application $app) {
+            return new BackupperFactory(
+                $app->make(DatabaseSessionRegistry::class),
+                $app->make(ConfigurationRegistry::class),
+                // Logger
+            );
+        });*/
+
+        /*$this->app->singleton(RestorerFactory::class, function (Application $app) {
+            return new RestorerFactory(
+                $app->make(DatabaseSessionRegistry::class),
+                $app->make(ConfigurationRegistry::class),
+                // Logger
+            );
+        });*/
+
+        /*$this->app->singleton(StatsProviderFactory::class, function (Application $app) {
+            return new StatsProviderFactory($app->make(DatabaseSessionRegistry::class));
+        });*/
+
+        /*$this->app
+            ->when([
+                AnonymizatorFactory::class,
+                BackupperFactory::class,
+                RestorerFactory::class,
+                StatsProviderFactory::class
+            ])
+            ->needs(LoggerInterface::class)
+            ->give(fn (Application $app) => $app->make('log'))
+        ;*/
+
+        $this->app
+            ->when([
+                AnonymizeCommand::class,
+                BackupCommand::class,
+                RestoreCommand::class,
+            ])
+            ->needs('$connectionName')
+            ->giveConfig('database.default')
+        ;
+        $this->app
+            ->when([
+                CheckCommand::class,
+                CleanCommand::class,
+                StatsCommand::class,
+            ])
+            ->needs('$defaultConnectionName')
+            ->giveConfig('database.default')
+        ;
+    }
+
+    /**
+     * Bootstrap any package services.
+     */
+    public function boot(): void
+    {
+        AboutCommand::add('DbToolsBundle', fn () => ['Version' => '2.0.0']);
```

Comment:
may be you should add laravel or something as dev dependencies

cf the static analysis failure:

![image](https://github.com/user-attachments/assets/cb582fb8-b044-45dd-b362-b8850749fa68)

Reply:
Exactly what I wanted to avoid :sweat_smile: But it apparently won't be inevitable :slightly_smiling_face: